src/_includes/install/windows.md

```diff
@@ -44,7 +46,27 @@ To upgrade the Dart SDK:
 C:\> choco upgrade dart-sdk
 ```
 
+### Uninstall using Chocolatey {:.no_toc}
+
+To uninstall the Dart SDK, use [Chocolatey][Chocolatey].
```

Comment:
This whole section is about using Chocolatey so I'd drop or modify this as it could be misunderstood by search or other tools as meaning its always used for uninstalling Dart.

Perhaps something like "To uninstall the Dart install managed by Chocolatey..."?